tests/gnrc_sock_tcp/main.c

```diff
@@ -33,6 +33,15 @@ void dump_args(int argc, char **argv)
     printf("\n");
 }
 
+void print_result(const char *name, int err)
+{
+    if (err) {
+        printf("%s: returns %s\n", name, strerror(err));
+    } else {
+        printf("%s: returns %d\n", name, err);
+    }
```

Comment:
No need for this. `strerror(0)` returns `"Success"`

Reply:
Not on every platform ;). And the python code calling the functions matches for the specific result. Thats why I handle 0 manually.

Reply:
BTW.: You added an implementation returning OK a few hours ago. :)

Reply:
arg sorry for that - feel free to change to return value to "Success" do you can get rid of this adapter function

Reply:
I prefer to keep it as it is. I am not sure if the returned string is always "Success" on all platforms. For any error code the return value should always be textual representation of the enum value but for 0 different strings are possible. Since this is test code I think keeping the tiny function is okay.